tests/test_bench.py

```diff
@@ -212,9 +193,11 @@ def test_quantumcircuit_indep_level(
     if scalable:
         assert qc.num_qubits == input_value
     assert benchmark.__name__.split(".")[-1] in qc.name
+    opt_level = 0
```

Comment:
```suggestion
    opt_level = 2
```
I think it makes sense to go with the Qiskit default here.